src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -258,6 +273,51 @@ void handlerReturnsInternalErrorWhenUriCreationFails() throws Exception {
         assertTrue(gatewayResponse.getBody().contains(ERROR_MESSAGE_SERVER_ERROR));
     }
 
+    @Test
+    void handlerReturnsCristinProjectsWhenQueryContainsPageParameter() throws Exception {
+        InputStream input = requestWithQueryParameters(
+            Map.of(TITLE, RANDOM_TITLE, LANGUAGE_QUERY_PARAMETER, LANGUAGE_NB, PAGE, PAGE_NUMBER_TWO));
+        handler.handleRequest(input, output, context);
+        GatewayResponse<ProjectsWrapper> gatewayResponse = GatewayResponse.fromOutputStream(output);
+
+        assertEquals(HttpURLConnection.HTTP_OK, gatewayResponse.getStatusCode());
+        assertTrue(gatewayResponse.getBody().contains(URI_WITH_PAGE_NUMBER_VALUE_OF_TWO));
+    }
+
+    @Test
+    void handlerThrowsBadRequestWhenQueryHasInvalidPageParameter() throws Exception {
+        InputStream input = requestWithQueryParameters(
+            Map.of(TITLE, RANDOM_TITLE, LANGUAGE_QUERY_PARAMETER, LANGUAGE_NB, PAGE, TITLE_ILLEGAL_CHARACTERS));
```

Comment:
There is some inconsistency with the parameters, they can be XXXXX, or XXXXX_KEY or XXX_QUERY_PARAMETER.